aladdin/__init__.py

```diff
@@ -1,2 +1,3 @@
-VERSION = (1, 19, 7, 0)
+VERSION = (1, 19, 7, 1)
```

Comment:
I guess I could leave it at `.0` too? since there's no code change, but not sure if that would ok with git tags and everything else, so I opted for revving to `.1`